Source/IGListCollectionView.m

```diff
@@ -13,7 +13,12 @@ @implementation IGListCollectionView
 
 - (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionViewLayout *)layout {
     if (self = [super initWithFrame:frame collectionViewLayout:layout]) {
-        self.backgroundColor = [UIColor whiteColor];
+        
+        UIColor *backgroundAppearanceColor = (UIColor *) [[[self class] appearance] backgroundColor];
```

Comment:
Actually, I had to add the cast to avoid a compile error.  At first I didn't have the cast when I had the sample project opened, but when I switched to the IGListKit project I got the following:

`Incompatible pointer types initializing 'UIColor *__strong' with an expression of type 'CGColorRef _Nullable' (aka 'struct CGColor *')`